src/ComponentPage.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+
+import './ComponentPage.scss'
+
+class ComponentPage extends Component {
+  renderSidebarLink(link, index) {
+    return (
+      <a
+        href={link.url}
+        key={index}
+        role='listitem'     
+        className={'mdc-list-item ' + link.className + (link.active ? ' sidebar-active': '')}>
```

Comment:
Are you familiar with the the es6 string concatenation?

`/`mdc-list-item ${link.className} ${link.active ? 'sidebar-active' : ''}/``

Reply:
No I didn't! Thanks 😀